finesse/hardware/plugins/temperature/senecak107.py

```diff
@@ -20,7 +20,17 @@ class SenecaK107Error(Exception):
     """Indicates that an error occurred while communicating with the device."""
 
 
-class SenecaK107(SerialDevice, TemperatureMonitorBase, description="Seneca K107"):
+class SenecaK107(
+    SerialDevice,
+    TemperatureMonitorBase,
+    description="Seneca K107",
+    parameters={
+        "min_temp": "The minimum temperature limit of the device",
+        "max_temp": "The maximum temperature limit of the device",
+        "min_millivolt": "The minimum voltage output (millivolts) of the device",
+        "max_millivolt": "The maximum voltage output (millivolts) of the device",
+    },
```

Comment:
Do we need to set values for these? cf. the equivalent for the DP9800 or TC4820, where the default baud rates are set.

Reply:
The default values are being defined in the `.__init__()` which is how the params are being defined in `em27_sensors.py` (the example Alex gave in #433).

Reply:
Ah yes, thanks for clarifying. I should have read the docs 😅